x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -62,17 +45,17 @@ export default ({ getService }: FtrProviderContext) => {
       });
 
       it('should delete transform by transformId', async () => {
-        const transformsInfo: TransformEndpointRequest[] = [{ id: transformId }];
+        const reqBody: DeleteTransformsRequestSchema = {
+          transformsInfo: [{ id: transformId, state: 'stopped' }],
```

Comment:
We should use the constants out of `TRANSFORM_STATE` in a lot of places in this file, rather than `stopped`, `started` etc.

Reply:
Fixed in e434c4f.